config/rbac/role_binding.yaml

```diff
@@ -8,5 +8,5 @@ roleRef:
   name: manager-role
 subjects:
 - kind: ServiceAccount
-  name: default
+  name: controller-manager
```

Comment:
this seems wrong to me, as in you shouldn't have to do it and it will overridden anyway.

Reply:
I would not like to use the `default` service account